### PR TITLE
Fixes typos and adds more talk tag lines

### LINKED ISSUE #66479: Yet another typo hunt
### Is your feature request related to a problem? Please describe.

Still crawling through the guts of the game and translating, and I found 10 minor typos/things that probably should be changed, somebody could get their first contribution fixing them right now, or just wait a few weeks until I inevitably find more, it's up to them.

### Solution you would like.

The typos I found so far, as well as the probable best fixes for them are as follow:

In the involuntary dimensional traveler dialogue, the player character's dialogue should probably capitalize English:
- I guess, there's no reason to expect a dimensional traveler to speak english. -> I guess, there's no reason to expect a dimensional traveler to speak English.

The description of the Optical Dampers is pretty unwieldy/out of the normal for CBM descriptions, a possible fix would be:
- The lenses surgically installed over your eyes can be tinted to block out incoming light. -> A pair of lenses surgically installed over your eyes that can be tinted to block out incoming light.

In the order acknowledgment NPC dialogue, this answer probably needs a period, but I could be wrong:
- Sure thing -> Sure thing.

In the monster special attacks:
- [monster] tries slam into [NPC name], but they dodge! -> [monster] tries to slam into [NPC name], but they dodge!

This semicolon in this specific weapon reaction to you bleeding is pretty awkward, and I guess it's debatable whether god should be capitalized:
- Oh God; Please don't die.  Not now! -> Oh God. Please don't die.  Not now! or Oh God, please don't die.  Not now!

Not exactly a typo per say, but the SIG MCX Rattler SBR's description mentioning it has a short barrel then just a few words later mentioning its barrel length is pretty clumsy writing, a potential fix could be:
- A short-barreled, gas-operated rifle, with a 5.5 inch barrel, built-in thin folding stock, and full rail mount on the upper receiver.  This particular one is a select-fire military version. -> A 5.5 inch, short-barreled, gas-operated rifle with built-in thin folding stock and full rail mount on the upper receiver.  This particular one is a select-fire military version.

Aftershock typos:
Attack messages for tusks and pointed tusks mutations, for both PCs and NPCs should have an exclamation mark at the end to bring them in line with other successful attack messages, as follows:
- You gore [target] with your tusks -> You gore [target] with your tusks!
- You stab [target] with your pointed tusks -> You stab [target] with your pointed tusks!
- [NPC name] gores [target] with their tusks -> [NPC name] gores [target] with their tusks!
- [NPC name] stabs [target] with their pointed tusks -> [NPC name] stabs [target] with their pointed tusks!

### Describe alternatives you have considered.

Not fixing typos?

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Procyonae**
#### Summary
None

#### Purpose of change

Fixes #66479

### CODE REVIEW

File: data/json/npcs/talk_tags.json [hunk 1]
```diff
@@ -6,11 +6,11 @@
     "text": [
       "Spirals.",
       "Spirals…",
-      "Spirals?",
+      "Spirals? <spirals>",
       "Spirals?  Spirals.",
       "Spiiiraals!",
       "Ha!  Spirals.",
-      "Spi…rals.",
+      "Spi…rals. <spirals>",
```
Comment (Maleclypse) [inline data/json/npcs/talk_tags.json:13]:
```suggestion
      "Spi…rals.  <spirals>",
```

File: data/json/npcs/talk_tags.json [hunk 2]
```diff
@@ -20,8 +20,14 @@
       "Spirals!!!",
       "SpIrAlS.",
       "SPIRALS.",
-      "AHAHAHAHA!!!",
-      "SpiRAAAAAAALS!"
+      "<spirals> AHAHAHAHA!!! <spirals>",
```
Comment (Maleclypse) [inline data/json/npcs/talk_tags.json:23]:
```suggestion
      "<spirals> AHAHAHAHA!!!  <spirals>",
```

File: data/json/npcs/talk_tags.json [hunk 3]
```diff
@@ -20,8 +20,14 @@
...
+      "<spirals> UEE HEE HEE! <spirals>",
```
Comment (Maleclypse) [inline data/json/npcs/talk_tags.json:24]:
```suggestion
      "<spirals> UEE HEE HEE!  <spirals>",
```

File: data/json/npcs/talk_tags.json [hunk 4]
```diff
@@ -20,8 +20,14 @@
...
+      "SpiRAAAAAAALS!",
+      "Spirals and spirals and spirals and spirals…",
+      "The spirals spiral into more spirals don't you see?",
+      "Now, now! Let the games begin! <spirals>",
```
Comment (Maleclypse) [inline data/json/npcs/talk_tags.json:28]:
```suggestion
      "Now, now! Let the games begin!  <spirals>",
```

File: data/json/npcs/talk_tags.json [hunk 5]
```diff
@@ -6,11 +6,11 @@
     "text": [
       "Spirals.",
       "Spirals…",
-      "Spirals?",
+      "Spirals? <spirals>",
```
Comment (Maleclypse) [inline data/json/npcs/talk_tags.json:9]:
```suggestion
      "Spirals?  <spirals>",
```

File: data/json/npcs/talk_tags.json [hunk 6]
```diff
@@ -20,8 +20,14 @@
...
       "SpIrAlS.",
       "SPIRALS.",
-      "AHAHAHAHA!!!",
-      "SpiRAAAAAAALS!"
+      "<spirals> AHAHAHAHA!!!  <spirals>",
+      "<spirals> UEE HEE HEE!  <spirals>",
+      "SpiRAAAAAAALS!",
+      "Spirals and spirals and spirals and spirals…",
+      "The spirals spiral into more spirals don't you see?",
+      "Now, now! Let the games begin!  <spirals>",
```
Comment (Procyonae) [inline data/json/npcs/talk_tags.json:28]:
```suggestion
      "Now, now!  Let the games begin!  <spirals>",
```

File: data/json/mutations/mutations.json [hunk 7]
```diff
@@ -7075,8 +7075,8 @@
     "category": [ "PLANT" ],
     "attacks": [
       {
-        "attack_text_u": "You lash %s with a vine",
-        "attack_text_npc": "%1$s lashes %2$s with their vines",
+        "attack_text_u": "You lash %s with a vine!",
```
Comment (NetSysFire) [inline data/json/mutations/mutations.json:7078]: Why does the player character use a single vine to attack while NPCs use multiple? Minor inconsistency. Same in line 7106.

**Review by NetSysFire — APPROVED** [review]
Terminal punctuation fixes, my favorite.

**Pull request merged**